Document/MediaManager.php

```diff
@@ -34,7 +34,7 @@ public function save($entity, $andFlush = true)
             $entity->setProviderName(func_get_arg(2));
         }
 
-        if ($andFlush && is_bool($andFlush)) {
+        if (is_bool($andFlush)) {
```

Comment:
I want convert one thousand articles with images. But I don't wont flush changes after each articles with media object. I want create media-object, save it w/o flushing. So I'm writing:
```php
// ... creating new media with image
$mediaManager->save($media, false);
$em->persist(); // may by it's excessive
// ... again
// ... again
// ... again
$em->flush();
```
But my `false` flag was ignored and all entities flushed after each saved media with article, authors, author pseudonyms etc. 
I think that this way is incorrect. The developer should be able to choose and this choice should be transparent